### DISCUSSION TITLE
Scroll content/index menu separate from article content

### DISCUSSION THREAD

**Discussion by tvarsis** [outline/outline#2349] [Ideas]
For larger articles with a lot of content and headings, the content menu becomes pretty useless since it is not possible to scroll in it, which leads to you only see the top part of it until you hit the bottom of the full article. And that pretty much removes the purpose of even having a content/index menu in the first place. I suggest that the content/index menu should have an independent scroll that can be used to scroll it without scrolling the actual article and vice versa.

**Comment by tommoor** [comment]
Hi, this was fixed [here](https://github.com/outline/outline/pull/2296) and will be in the next build towards the start of August.

**Reply by tvarsis** [reply]
Fantastic news! Thanks!
